gradle.properties

```diff
@@ -1,3 +1,4 @@
 kotlin.code.style=official
 kotlin.js.compiler=ir
 kotlin.daemon.jvmargs=-Xmx4G
+kotlin.js.yarn=false
```

Comment:
I had to quit yarn to use jsr packages (Kotlin/JS does not recognize .npmrc with yarn)

Reply:
Okay let's just use npm. I'm always uncomfortable with the yarn.lock file as well 😂 
Can we remove those yarn-specific files?

Reply:
I deleted `yarn.lock` but it spawn again somehow...

Reply:
Somehow it started to work, so I'm reverting this change.